wordpress-mobile/WordPress-Android
# Issue/5030 add aztec setting

### Fix
Add option in App Settings to select between Aztec, Visual, and Legacy editors as described in https://github.com/wordpress-mobile/WordPress-Android/issues/5030.

### Test
1. Go to ***Me*** tab.
2. Tap ***App Settings*** option.
3. Tap ***Set editor type*** option.
4. Tap ***Aztec*** option.
5. Go to ***My Sites*** tab.
6. Tap ***Create*** button.
7. Notice ***Tenochtitlan is the hint*** as the hint.
8. Tap back arrow in the action bar.
9. Go to ***Me*** tab.
10. Tap ***App Settings*** option.
11. Tap ***Set editor type*** option.
12. Tap ***Visual*** option.
13. Go to ***My Sites*** tab.
14. Tap ***Create*** button.
15. Notice ***Share your story here...*** as the hint.
16. Tap back arrow in the action bar.
17. Go to ***Me*** tab.
18. Tap ***App Settings*** option.
19. Tap ***Set editor type*** option.
20. Tap ***Legacy*** option.
21. Go to ***My Sites*** tab.
22. Tap ***Create*** button.
23. Notice ***Content (tap to add text and media)*** as the hint.

#### Note
This setting is meant to test the Aztec editor only.  This should be removed once testing is complete.
